Integrations/EWSMailSender/EWSMailSender.py

```diff
@@ -138,6 +149,18 @@ def send_email(to, subject, body="", bcc=None, cc=None, replyTo=None, htmlBody=N
         with open(file_path, 'rb') as f:
             attachments.append(FileAttachment(content=f.read(), name=attachment_name))
 
+    for i in range(0, len(file_entries_for_attachments_inline)):
+        entry_id = file_entries_for_attachments_inline[i]
+        attachment_name_inline = attachments_names_inline[i]
+        try:
+            res = demisto.getFilePath(entry_id)
+        except Exception as ex:
+            raise Exception("entry %s does not contain a file" % entry_id)
+        file_path = res["path"]
```

Comment:
why do you separate this loop from the previous one (132-139)? you already run the `getFilePath`

Reply:
I run getFilePath on different lists. The second list includes list with item-ids that will be attached as inline. I append to attachment list with `attachments.append(FileAttachment(content=f.read(), name=attachment_name_inline, is_inline=True, content_id=attachment_name_inline))` and in first loop I append ` attachments.append(FileAttachment(content=f.read(), name=attachment_name))`. Need to separate list with attachIDs and attachCIDs as one is attached as regular attachment and the second is will be attached inline.

Reply:
you are looking at different loop. look at the lines i specified.